data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/RdsSourceConfig.java

```diff
@@ -21,6 +24,7 @@
  * Configuration for RDS Source
  */
 public class RdsSourceConfig {
+    private static final int DEFAULT_S3_FOLDER_PARTITION_COUNT = 100;
```

Comment:
Is the same default value for both export and stream ?

Reply:
Yes